### PR TITLE
Doc: Rpi OS Setup

### PR DESCRIPTION
**Pull request by efernandezec**

### CODE REVIEW

File: content/en/docs/guides/Getting Started/Bringup/index.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+title: "Bringup Mardan Robot"
+linktitle: "Bringup"
+date: 2023-07-21
+weight: 2
+collapsible: false
+---
+
+{{< alert title="Bringup" >}}
+The code will be extracted from the [MardanRobotRpi](https://github.com/roboticamed/MardanRobotRpi) GitHub repository.
```
Comment (jadarve) [inline content/en/docs/guides/Getting Started/Bringup/index.md]: Maybe change "will" by "is"

File: content/en/docs/guides/Getting Started/Bringup/index.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+The code will be extracted from the [MardanRobotRpi](https://github.com/roboticamed/MardanRobotRpi) GitHub repository. 
+{{< /alert >}}
+
+### 1. Install Docker Compose  into RPi 4
+``` shell
+sudo apt-get install -y libffi-dev libssl-dev python3-dev python3 python3-pip
+
+sudo pip3 install docker-compose
+
+sudo systemctl enable docker
```
Comment (jadarve) [inline content/en/docs/guides/Getting Started/Bringup/index.md]: Isn't docker already enabled from the rpi_setup document? If not, we should add these instructions there.

File: content/en/docs/guides/Getting Started/Bringup/index.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+weight: 2
+collapsible: false
+---
+
+{{< alert title="Bringup" >}}
+The code will be extracted from the [MardanRobotRpi](https://github.com/roboticamed/MardanRobotRpi) GitHub repository. 
+{{< /alert >}}
+
+### 1. Install Docker Compose  into RPi 4
+``` shell
```
Comment (jadarve) [inline content/en/docs/guides/Getting Started/Bringup/index.md]: Missing blank line between Heading and paragraph, for this section and all below.

File: content/en/docs/guides/Getting Started/Bringup/index.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+Run  Docker Compose in the directory ~/MardanRobotRpi/RPi.
+```shell
+docker-compose up
+```
+That should give an output similar to:
+
+![](docker-compose.png)
+
+>**Note:** Be sure that ROS master is connected
```
Comment (jadarve) [inline content/en/docs/guides/Getting Started/Bringup/index.md]: How?

File: content/en/docs/guides/Getting Started/_index.md [hunk 5]
```diff
@@ -0,0 +1,6 @@
+---
```
Comment (jadarve) [inline content/en/docs/guides/Getting Started/_index.md]: Better if we don't use spaces in the folder names, that messes up navigation in a terminal :grimacing: 

Let's use snake_case instead.

File: content/en/docs/guides/setup/Possible Error Solutions/index.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
+---
+title: "Possible Error Solutions"
+linktitle: "Possible Error Solutions"
+date: 2023-07-21
+weight: 1
```
Comment (jadarve) [inline content/en/docs/guides/setup/Possible Error Solutions/index.md]: There is already a weight 1 document, the OS preparation.

File: content/en/docs/guides/setup/Possible Error Solutions/index.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
+---
```
Comment (jadarve) [inline content/en/docs/guides/setup/Possible Error Solutions/index.md]: Same with the folder title :grimacing:  Also, perhaps another title such as "Known problems" or "Troubleshooting" can be more meaningful.

Regarding this page's content. What do you think about having it as part of the rpi4_setup document?

File: content/en/docs/guides/setup/rpi4_setup/index.md [hunk 8]
```diff
@@ -75,8 +148,8 @@ In the Raspberry terminal, run:
 curl -fsSL https://get.docker.com -o get-docker.sh
 sudo sh get-docker.sh
 
-# Grant priviledges to the rospi user to run docker containers
-sudo usermod -aG docker raspi
+# Grant priviledges to the Rpi Username  that was seted during the extra setting Raspberry Pi  , to run docker containers
```
Comment (jadarve) [inline content/en/docs/guides/setup/rpi4_setup/index.md]: Typo in seted, should be "set".

Also, extra space before a comma.

File: content/en/docs/guides/setup/rpi4_setup/index.md [hunk 9]
```diff
@@ -1,5 +1,5 @@
 ---
```
Comment (jadarve) [inline content/en/docs/guides/setup/rpi4_setup/index.md:1]: This is better documentation than that in #6. Let's keep this one :tada:

File: content/en/docs/guides/Getting Started/Bringup/index.md (hunk 2 above)
Comment (aldajo92) [reply content/en/docs/guides/Getting Started/Bringup/index.md]: solved

File: content/en/docs/guides/Getting Started/Bringup/index.md (hunk 3 above)
Comment (aldajo92) [reply content/en/docs/guides/Getting Started/Bringup/index.md]: Solved

File: content/en/docs/guides/Getting Started/_index.md (hunk 5 above)
Comment (aldajo92) [reply content/en/docs/guides/Getting Started/_index.md]: Solved

File: content/en/docs/guides/setup/Possible Error Solutions/index.md (hunk 6 above)
Comment (aldajo92) [reply content/en/docs/guides/setup/Possible Error Solutions/index.md]: Solved

File: content/en/docs/guides/setup/rpi4_setup/index.md (hunk 9 above)
Comment (aldajo92) [reply content/en/docs/guides/setup/rpi4_setup/index.md:1]: Solved

File: content/en/docs/guides/Getting Started/Bringup/index.md (hunk 1 above)
Comment (aldajo92) [reply content/en/docs/guides/Getting Started/Bringup/index.md]: Solved

File: content/en/docs/guides/Getting Started/Bringup/index.md (hunk 4 above)
Comment (aldajo92) [reply content/en/docs/guides/Getting Started/Bringup/index.md]: Solved

File: content/en/docs/guides/setup/Possible Error Solutions/index.md (hunk 7 above)
Comment (aldajo92) [reply content/en/docs/guides/setup/Possible Error Solutions/index.md]: Solved

File: content/en/docs/guides/setup/rpi4_setup/index.md (hunk 8 above)
Comment (aldajo92) [reply content/en/docs/guides/setup/rpi4_setup/index.md]: Solved

**Pull request merged**